cypress/tests/e2e/a-setup.cy.ts

```diff
@@ -25,6 +34,11 @@ xdescribe('Prepare the cluster for test', () => {
     });
   });
 
+  it('close welcome modal if it appears', () => {
+    cy.wait(15000);
```

Comment:
do we need a separate test/action? it seems we are embedding `tickWelcomeModal()` in the tests anyway

Reply:
I'm trying to get the test pass, the welcome modal is appearing in later test and it's odd.